test/test_plugins.py

```diff
@@ -243,9 +243,8 @@ def import_task_created_event(self, session, task):
 
         logs = [line for line in logs if not line.startswith('Sending event:')]
         self.assertEqual(logs, [
-            'Album: {0}/album'.format(self.import_dir),
-            '  {0}'.format(self.file_paths[0]),
-            '  {0}'.format(self.file_paths[1]),
```

Comment:
Yes, I did...
Maybe, I got you wrong, but there is a plugin loaded just a few lines before. The plugin which is loaded there converts every `AlbumImportTask` into a set of `SingletonImportTask`s. Since the return values of the plugins were not used until my change, the test failed after doing the change.
But I guess, I should also add a test without any plugin loaded to check the normal behaviour.